exercises/practice/grains/.approaches/max-value/snippet.txt

```diff
@@ -0,0 +1,4 @@
+public static ulong Total()
+{
+  return System.UInt64.MaxValue;
```

Comment:
```suggestion
    return System.UInt64.MaxValue;
```

Reply:
It kills me that sometimes I get two-space indentation and sometimes four. Haven't kept track of where they differ.